.github/workflows/release.yaml

```diff
@@ -60,14 +65,33 @@ jobs:
         push: true
         cache-from: type=gha
         cache-to: type=gha,mode=max
+        provenance: false
+        sbom: false
```

Comment:
Non blocking, but any reason we are not publishing this? It is slowly becoming an expected practice.

Reply:
`provenance: false`  : 
Is not a real true provenance and cannot be attested to, we use `slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@v2.0.0` which also does not attach an unknown artifact to the image.

`sbom: false`  We are using `anchore/sbom-action` for SBOMS